service-api/src/main/java/greencity/dto/comment/AddCommentDtoRequest.java

```diff
@@ -0,0 +1,18 @@
+package greencity.dto.comment;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+import org.hibernate.validator.constraints.Length;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode
+public class AddCommentDtoRequest {
+    @NotBlank(message = "The text of comment can not be empty")
```

Comment:
please update message  to follow `NotBlank` documentation
`The annotated element must not be null and must contain at least one non-whitespace character. Accepts CharSequence.`